ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -189,32 +201,85 @@ message DamlPackageUploadRejectionEntry {
     string details = 1;
   }
 
-  // The participant was not authorized to submit the upload request.
-  message ParticipantNotAuthorized {
-    string details = 1;
-  }
-
   oneof reason {
-
     InvalidPackage invalid_package = 3;
     ParticipantNotAuthorized participant_not_authorized = 4;
   }
 }
 
-message DamlConfigurationEntry {
-  DamlTimeModel time_model = 1;
+// Configuration change request to change the ledger configuration.
+message DamlConfigurationSubmission {
```

Comment:
It seems configuration needs a submission id, just like party allocation and package upload, else it is difficult to match responses with requests, generation number is not globally unique and 2 participants will decide on the same subsequent number, opening up the possibility of mixing up the responses.

Reply:
I considered it, but felt that it doesn't need to be that complex due to the configuration changes being very rare and almost invariably coming from a single source (by design configuration can also only be changed by one "admin" participant). Though on the other hand if we add things to the configuration that may be tweaked more regularly or from multiple sources at once we might want to consider just adding it in. 

Do you think we still should add it?

Reply:
I would add it for consistency sake

Reply:
cool, I'll add it.